### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_Comparison_ScreenReader: Status message not announced for the data that is updated on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9474]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#MacOs;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev1;#MAS4.1.3;#Voiceover;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
MAS reference: MAS4.1.3
Mac OS:10.15.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Check if status message is announced for all the updated content on the screen by Voice over.

**Expected:**
All the data that gets updated on the screen should be announced by the screen reader.

**User Impact:**
Screen reader users will not receive all the information that is updated on the screen.

**Attachment link reference:**
[9474_A11y_SQLAzureDataStudio_SchemaCompare_Comparison_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4297121/9474_A11y_SQLAzureDataStudio_SchemaCompare_Comparison_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
dupe of #9401 
@v-atkulk  please coordinate with your team members and don't create duplicates, we have seen a lot of duplicate issues being created recently.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-Win32-Feb2020;
